src/frontend/efiling-frontend/src/domain/package/package-review/DocumentList.js

```diff
@@ -1,9 +1,25 @@
 import React from "react";
 import PropTypes from "prop-types";
 
+import { errorRedirect } from "../../../modules/helpers/errorRedirect";
+import { getSubmittedDocument } from "./PackageReviewService";
 import "./DocumentList.css";
 
-export default function DocumentList({ documents }) {
+export default function DocumentList({ packageId, documents }) {
+  function handleClickFile(document) {
+    getSubmittedDocument(packageId, document).catch((err) => {
```

Comment:
That looks really good, is it the service suppose to save the file locally or should it return the blob, don't want to change it now, but something to thing about. or the name of the function should be `saveSubmittedDocumentLocal` or something like that, what do you think ?

Reply:
Good suggestion.  Refactored both this method and getSubmissionSheet to downloadSubmittedDocument and downloadSubmissionSheet respectively.